src/rulesets/headers.ts

```diff
@@ -5,13 +5,12 @@ import { links } from "../docs";
 
 export const rules = {
   headerNameCase: ({ responses }: SnykApiCheckDsl) => {
-    responses.headers.requirementOnChange.must(
-      "be kebab-case",
-      ({ name }, context, docs) => {
+    responses.headers.requirementOnChange
+      .attributes("name")
```

Comment:
constrain the header naming rule to changes to its name

**Currently** if you changed the description of a header with an invalid name, the rule would not pass. It's a breaking change to fix a name leading to a catch 22. 

**Now** all the sibling attributes of `name` can change without triggering this rule